### PR TITLE
Add relations table for samples

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-1416

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java [hunk 1]
```diff
@@ -741,6 +741,15 @@ public ModelAndView setupForm(@PathVariable Long sampleId, @PathVariable Long pr
         model.put("poolSampleMap", poolSampleMap);
         model.put("samplePools", pools);
         model.put("sampleRuns", getRunsBySamplePools(pools));
+        List<SampleDto> relations = new ArrayList<>();
+        if (sample instanceof DetailedSample) {
```
Comment (apmasell) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java]: Fixed.

**Comment by apmasell** [comment]
It probably should include libraries and dilutions, but we don't have a consistent DTO for those unless they all get converted to boxables or pool views, which strips out the sample class. I guess we could make a new DTO type for this. Thoughts?

The library error is unrelated, but I'll fix it.

**Comment by djcooke** [comment]
Yeah, a "SampleRelativeDto" makes sense

**Comment by apmasell** [comment]
Okay. Let's go with this as a stop-gap and I'll solicit what the lab wants in such a table.

**Comment by apmasell** [comment]
Mentioned in https://jira.oicr.on.ca/browse/GLT-253

**Comment by tonydebat** [comment]
This sample takes 24 seconds to load. (Has a huge number of children, not typical.)
http://localhost:8080/miso/sample/40229

**Pull request merged**